src/new.spec.js

```diff
@@ -234,4 +234,60 @@ describe("withStore", () => {
       });
     });
   });
+
+  describe("effects", () => {
+    it("receives the passed arguments", () => {
+      const args = ["bar", "baz"];
+      const { effects } = makeTestInstance({
+        effects: {
+          foo: (...rest) => {
```

Comment:
Let's avoid the arrow syntax for effects because it does not make sense as they cannot access the store via `this`.